Makefile.core.mk

```diff
@@ -72,6 +72,7 @@ ENVTEST_K8S_VERSION ?= 1.29.0
 DOCKER_BUILD_FLAGS ?= "--platform=$(TARGET_OS)/$(TARGET_ARCH)"
 
 VERBOSE_FLAG := $(if $(VERBOSE),-v)
+NOCOLOR_FLAG := $(if [ -t 1 ],--no-color,)
```

Comment:
I'd rather join these two into a single flag, sth like `TEST_FLAGS`

Reply:
We could have two vars: `GOTEST_FLAGS` and `GINKGO_FLAGS`, but would that really be better than what we have now?

Reply:
I'd personally prefer `GOTEST_FLAGS` and `GINKGO_FLAGS` but it's not important

Reply:
Argh. I tested this and remember seeing it work correctly both when attached and when not attached, but when I tested it again, it doesn't behave correctly.

It turns out that when you run something in `$(cmd)`, it's never attached to a terminal. I can't get it to work, even when I add `$$( [ -t 1 ] ...)` when invoking the ginkgo command. The only two solutions that work are:
- have the make target call a shell script, which can then check whether it's attached
- use the CI env var instead of checking whether the terminal is attached

I don't like either solution. @dgn @jwendell your thoughts?

Reply:
check the `CI` env var.

We can always come up with better solutions when they are needed. Let's just fix this issue and move on